libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -606,18 +680,53 @@ bool AP_DDS_Client::create()
             requests[2] = dreader_req_id;
 
             if (!uxr_run_session_until_all_status(&session, requestTimeoutMs, requests, status, nRequests)) {
-                GCS_SEND_TEXT(MAV_SEVERITY_ERROR,"XRCE Client: Topic/Sub/Reader session request failure for index '%d'",(int)i);
+                debug("XRCE Client: Topic/Sub/Reader session request failure for index '%u'",i);
                 for (uint8_t s = 0 ; s < nRequests; s++) {
-                    GCS_SEND_TEXT(MAV_SEVERITY_ERROR,"XRCE Client: Status '%d' result '%u'", s, status[s]);
+                    debug("XRCE Client: Status '%d' result '%u'", s, status[s]);
```

Comment:
Just curious - what is the reason to make these optional debug messages, all at the same level?

Reply:
@Ryanf55 tridge requested for the session failure messages for topics and services to be made as debug messages .. so to keep things in line I made the status messages as debug

Reply:
Ok, I'll follow up with Tridge - Errors would now be silently swallowed, but other libraries are allowed to call `GCS_SEND_TEXT(MAV_SEVERITY_ERROR` during initialization.